README.md

```
@@ -14,50 +14,54 @@ _Organize ideas and collaborate using Markdown, a lightweight language for text
 
-<!-- For start course, run in JavaScript:
-'https://github.com/new?' + new URLSearchParams({
-  template_owner: 'skills',
-  template_name: 'communicate-using-markdown',
-  owner: '@me',
-  name: 'skills-communicate-using-markdown',
-  description: 'My clone repository',
-  visibility: 'public',
-}).toString()
--->
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)